--- gp_help.xml	2008-04-15 21:02:39.000000000 +1000
+++ gp_help.xml.fixed	2008-04-15 21:02:21.000000000 +1000
@@ -309,7 +309,7 @@
 f(x) = a*x+b
 fit f(x) 'data.dat' index 1 using 2:3 via a,b
 
-The coefficients to be varied are listed after the via keyword; the modifiers index, every and using have the same meanings as in the plot command. This is useful for producing best-fit lines and also has applications for estimating the gradients of datasets. The syntax is essentially identical to that used by Gnuplot, though a few points, outlined in Section, are worth noting. 
+The coefficients to be varied are listed after the via keyword; the modifiers index, every and using have the same meanings as in the plot command. This is useful for producing best-fit lines and also has applications for estimating the gradients of datasets. The syntax is essentially identical to that used by Gnuplot.
 
   </fit>
   <help>
@@ -423,7 +423,7 @@
 
 plot 'data.dat' with points
 
-which would plot the file called `data.dat'. Section  should be studied for further details of the format that is expected of input data files, and how PyXPlot may be directed to plot only certain portions of data files. Multiple datasets can be plotted on a single graph by listing them with commas separating them: 
+which would plot the file called `data.dat'. Multiple datasets can be plotted on a single graph by listing them with commas separating them: 
 
 plot sin(x) with colour blue, cos(x) with linetype 2
 
@@ -539,7 +539,7 @@
 set key###|#Set option ON#\\
 set nokey#|#Set option OFF
 
-More details of the functions of each individual setting can be found in the subsections below, which forms a complete list of the recognised setting keywords. The reader should also see the show command, which can be used to display the current values of settings, and the unset command, which returns settings to their default values. Section describes how commonly used settings can be saved into a configuration file. 
+More details of the functions of each individual setting can be found in the subsections below, which forms a complete list of the recognised setting keywords. The reader should also see the show command, which can be used to display the current values of settings, and the unset command, which returns settings to their default values. 
    <arrow>
 
 set arrow \labarrow number\rab from [\labco-ordinate\rab] \labx\rab,
@@ -588,7 +588,7 @@
 
 set axescolour blue
 
-changes the axes to be blue. Any of the recognised colour names listed in Section can be used. 
+changes the axes to be blue.
 
    </axescolour>
    <axis>
@@ -755,7 +755,7 @@
 
 set gridmajcolour purple
 
-would cause the major grid lines to be drawn in purple. Any of the recognised colour names listed in Section can be used. See also the set gridmincolour command. 
+would cause the major grid lines to be drawn in purple. See also the set gridmincolour command. 
 
    </gridmajcolour>
    <gridmincolour>
@@ -766,7 +766,7 @@
 
 set gridmincolour purple
 
-would cause the minor grid lines to be drawn in purple. Any of the recognised colour names listed in Section can be used. 
+would cause the minor grid lines to be drawn in purple. 
 
 See also the set gridmajcolour command.
 
@@ -906,7 +906,7 @@
 
 set multiplot
 
-causes PyXPlot to enter multiplot mode, which allows many graphs to be plotted together and displayed side-by-side. See Section for a full discussion of multiplot mode. 
+causes PyXPlot to enter multiplot mode, which allows many graphs to be plotted together and displayed side-by-side.
 
    </multiplot>
    <mxtics>
@@ -1072,7 +1072,7 @@
 
 plot sin(x), cos(x), tan(x), exp(x)
 
-the first function would be plotted in red, the second in green, and the third in blue. Upon reaching the fourth, the palette would cycle back to red. Any of the recognised colour names listed in Section can be used. 
+the first function would be plotted in red, the second in green, and the third in blue. Upon reaching the fourth, the palette would cycle back to red. 
 
    </palette>
    <papersize>
@@ -1085,7 +1085,136 @@
 set papersize letter
 set papersize 200,100
 
-A list of recognised papersizes can be found in Figure. 
+The following named papersizes are recognised:
+
+|###########################2a0###1681###1189
+|###########################4a0###2378###1681
+|############################a0###1189####840
+|############################a1####840####594
+|###########################a10#####37#####26
+|############################a2####594####420
+|############################a3####420####297
+|############################a4####297####210
+|############################a5####210####148
+|############################a6####148####105
+|############################a7####105#####74
+|############################a8#####74#####52
+|############################a9#####52#####37
+|############################b0###1414####999
+|############################b1####999####707
+|###########################b10#####44#####31
+|############################b2####707####499
+|############################b3####499####353
+|############################b4####353####249
+|############################b5####249####176
+|############################b6####176####124
+|############################b7####124#####88
+|############################b8#####88#####62
+|############################b9#####62#####44
+|############################c0###1296####917
+|############################c1####917####648
+|###########################c10#####40#####28
+|############################c2####648####458
+|############################c3####458####324
+|############################c4####324####229
+|############################c5####229####162
+|############################c6####162####114
+|############################c7####114#####81
+|############################c8#####81#####57
+|############################c9#####57#####40
+|#########################crown####508####381
+|##########################demy####572####445
+|###################double_demy####889####597
+|######################elephant####711####584
+|###################envelope_dl####110####220
+|#####################executive####267####184
+|######################foolscap####330####203
+|#############government_letter####267####203
+|####international_businesscard#####85#####53
+|###################japanese_b0###1435###1015
+|###################japanese_b1###1015####717
+|##################japanese_b10#####44#####31
+|###################japanese_b2####717####507
+|###################japanese_b3####507####358
+|###################japanese_b4####358####253
+|###################japanese_b5####253####179
+|###################japanese_b6####179####126
+|###################japanese_b7####126#####89
+|###################japanese_b8#####89#####63
+|###################japanese_b9#####63#####44
+|################japanese_kiku4####306####227
+|################japanese_kiku5####227####151
+|#############japanese_shiroku4####379####264
+|#############japanese_shiroku5####262####189
+|#############japanese_shiroku6####188####127
+|####################large_post####533####419
+|########################ledger####432####279
+|#########################legal####356####216
+|########################letter####279####216
+|########################medium####584####457
+|#######################monarch####267####184
+|##########################post####489####394
+|#####################quad_demy###1143####889
+|########################quarto####254####203
+|#########################royal####635####508
+|#####################statement####216####140
+|####################swedish_d0###1542###1090
+|####################swedish_d1###1090####771
+|###################swedish_d10#####48#####34
+|####################swedish_d2####771####545
+|####################swedish_d3####545####385
+|####################swedish_d4####385####272
+|####################swedish_d5####272####192
+|####################swedish_d6####192####136
+|####################swedish_d7####136#####96
+|####################swedish_d8#####96#####68
+|####################swedish_d9#####68#####48
+|####################swedish_e0###1241####878
+|####################swedish_e1####878####620
+|###################swedish_e10#####38#####27
+|####################swedish_e2####620####439
+|####################swedish_e3####439####310
+|####################swedish_e4####310####219
+|####################swedish_e5####219####155
+|####################swedish_e6####155####109
+|####################swedish_e7####109#####77
+|####################swedish_e8#####77#####54
+|####################swedish_e9#####54#####38
+|####################swedish_f0###1476###1044
+|####################swedish_f1###1044####738
+|###################swedish_f10#####46#####32
+|####################swedish_f2####738####522
+|####################swedish_f3####522####369
+|####################swedish_f4####369####261
+|####################swedish_f5####261####184
+|####################swedish_f6####184####130
+|####################swedish_f7####130#####92
+|####################swedish_f8#####92#####65
+|####################swedish_f9#####65#####46
+|####################swedish_g0###1354####957
+|####################swedish_g1####957####677
+|###################swedish_g10#####42#####29
+|####################swedish_g2####677####478
+|####################swedish_g3####478####338
+|####################swedish_g4####338####239
+|####################swedish_g5####239####169
+|####################swedish_g6####169####119
+|####################swedish_g7####119#####84
+|####################swedish_g8#####84#####59
+|####################swedish_g9#####59#####42
+|####################swedish_h0###1610###1138
+|####################swedish_h1###1138####805
+|###################swedish_h10#####50#####35
+|####################swedish_h2####805####569
+|####################swedish_h3####569####402
+|####################swedish_h4####402####284
+|####################swedish_h5####284####201
+|####################swedish_h6####201####142
+|####################swedish_h7####142####100
+|####################swedish_h8####100#####71
+|####################swedish_h9#####71#####50
+|#######################tabloid####432####279
+|###############us_businesscard#####89#####51
 
    </papersize>
    <pointlinewidth>
@@ -1133,6 +1262,7 @@
 set size (\labwidth\rab|ratio \labratio\rab|noratio|square)
 
 The setting size is deprecated; use set width instead. It sets the width of the plot in centimetres. However, the command set size, when followed by the keyword ratio, is still used to set the aspect ratio of plots. See the `ratio' setting below for details. 
+
     <noratio>
 
 set size noratio
@@ -1305,7 +1435,7 @@
 
 set textcolour red
 
-causes all text labels, including the labels on graph axes and legends, etc. to be rendered in red. Any of the recognised colour names listed in Section can be used, as can a number which indexes into the current palette. 
+causes all text labels, including the labels on graph axes and legends, etc. to be rendered in red.
 
    </textcolour>
    <texthalign>
@@ -1504,7 +1634,7 @@
 set output 'filtered.dat'
 tabulate 'data.dat' u 3:6:9 select (asin($4)\rab0)
 
-The format used in each column of the output file is chosen automatically with integers and small numbers treated intelligently to produce output which preserves accuracy, but is also easily human-readable. If desired, however, a format statement may be specified using the with format specifier. The syntax for this is similar to that expected by the Python string substitution operator (%)[*]. For example, to tabulate the values of x to very many significant figures one could use: 
+The format used in each column of the output file is chosen automatically with integers and small numbers treated intelligently to produce output which preserves accuracy, but is also easily human-readable. If desired, however, a format statement may be specified using the with format specifier. The syntax for this is similar to that expected by the Python string substitution operator (%). For example, to tabulate the values of x to very many significant figures one could use: 
 
 tabulate x**2 with format "%27.20e"
 
@@ -1512,7 +1642,7 @@
 
 tabulate [0:2*pi] sin(x):cos(x):tan(x) u 1:2:3:4
 
-The samples setting can be used to control the number of points that are inserted into the data file. If the x-axis is set to be logarithmic then the points at which the functions are evaluated are spaced logarithmically. * -- Note that this operator can also be used within PyXPlot; see Section for details.
+The samples setting can be used to control the number of points that are inserted into the data file. If the x-axis is set to be logarithmic then the points at which the functions are evaluated are spaced logarithmically.
 
   </tabulate>
   <text>
